COLLABORATOR_GUIDE.md

```diff
@@ -519,9 +501,26 @@ Save the file and close the editor. You'll be asked to enter a new
 commit message for that commit. This is a good moment to fix incorrect
 commit logs, ensure that they are properly formatted, and add
 `Reviewed-By` lines.
+
 * The commit message text must conform to the
 [commit message guidelines](./CONTRIBUTING.md#commit-message-guidelines).
 
+* Always modify the original commit message to include additional meta
```

Comment:
How about mentioning `node-review` extension here? It doesn't make sense to even try to extract this info manually.

Reply:
We already mention `node-review` in [the onboarding doc](https://github.com/nodejs/node/blob/f3eb193a3019d79a82cec8b561e028e4e37a7b87/doc/onboarding.md#exercise-make-a-pr-adding-yourself-to-the-readme), though I'm not really sure how to reduce duplication here.

+1 on mentioning `node-review` here.

Reply:
Great idea. I added a sentence and link for `node-review`. I also updated the contents of the onboarding doc to note that `node-review` works in Edge browser too. At some point soon, we'll have to add the CLI tool for this that @joyeecheung et al. have been working on.